PocketBeagle/pru/ring.pru0.c

```diff
@@ -1,49 +1,39 @@
-/*
- *
- * Build and run with:
- * make -C /var/lib/cloud9/examples/extras/pru PRUN=pru0 TARGET=ring
```

Comment:
Confused a bit as I recall changing the path. How is this on old code? What did I miss?

Reply:
Hmmmm.... my copy has the comment removed since it works with the standard Makefile.